### PR TITLE
eth/sync: augment log with peer count information

### PR DESCRIPTION
**Pull request by q9f**
## PR description

I'm running a lot of Besu nodes and it bothered me that there is only a little information about networking available in the default `INFO` log level. This pull request tries to improve the logging a bit.

- augment `LOG.info()` with `peerCount` wherever sensible in `eth/sync/*`
- shortened block hashes to decrease the required terminal width, e.g., `0x3b64..5bcb`
- fixed some minor wording issues

Screenshot at 2020-04-01 15-19-05

Screenshot at 2020-04-01 15-37-29

Screenshot at 2020-04-01 17-13-17

Submitting this as draft PR. I'm not trying to blow up complexity but passing `ethContext` to the import steps is breaking some tests. I tried to investigate but I couldn't find an easy way to provide an `ethContext` in the block import test cases. I'm not very familiar with the code-base yet. Any pointers appreciated. :pray: 

## Fixed Issue(s)
N/A

### CODE REVIEW

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java [hunk 1]
```diff
@@ -341,16 +341,23 @@ private void broadcastBlock(final Block block, final BlockHeader parent) {
                     block.getHash());
               } else {
                 final double timeInS = importTask.getTaskTimeInSec();
+                final String blockHash = block.getHash().toHexString();
+                final String shortHash =
+                    String.format(
```
Comment (shemnon) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java:346]: My concern with short hashes is then we cannot go to etherscan from the log.  I do this fairly frequently when analyzing short lived forks.

Perhaps put the peers not at the end and making the other numbers fixed width?

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fullsync/FullImportBlockStep.java [hunk 2]
```diff
@@ -30,23 +31,37 @@
...
+            blockHash.substring(blockHash.length() - 4, blockHash.length()));
     final BlockImporter<C> importer =
         protocolSchedule.getByBlockNumber(blockNumber).getBlockImporter();
     if (!importer.importBlock(protocolContext, block, HeaderValidationMode.SKIP_DETACHED)) {
       throw new InvalidBlockException("Failed to import block", blockNumber, block.getHash());
     }
     if (blockNumber % 200 == 0) {
-      LOG.info("Import reached block {}", blockNumber);
+      LOG.info(
+          "Import reached block {} ({}), Peers: {}",
```
Comment (shemnon) [inline ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fullsync/FullImportBlockStep.java:61]: short hash here is fine.

File: ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java (hunk 1 above)
Comment (q9f) [reply ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/BlockPropagationManager.java:346]: What about printing out an additional debug level log info with the full hash? My impression is that the full details are too much on info log level.

**Comment by q9f** [comment]
Ok, restored the full block hash :blush:

**Comment by q9f** [comment]
Friendly Monday morning bump :coffee: :pizza: :blush:

**Comment by shemnon** [comment]
Interestingly enough Git won't let me auto merge.  @q9f can you push up a master merge?  hopefully that will unstick it.

**Comment by q9f** [comment]
Done :)

**Pull request merged**